openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -704,12 +639,34 @@ def build_anim(project_name, asset_name):
     errors = []
     setdress_container = None
     for container in bpy.context.window_manager.openpype_containers:
-        container_metadata = container["avalon"]
+        container_metadata = container.get("avalon", {})
+        if not container_metadata:
+            continue
+
         family = container_metadata.get("family")
 
         if family not in {"rig", "model", "setdress"}:
             continue
 
+        if family == "rig":
+            # Download lip animation
+            lip_anim_repre = download_subset(
```

Comment:
Don't we want `hero` versions?

Reply:
I'm not sure if that's beneficial. Animators may override the lip action and with the workflow we established updating the lip action won't automatically update the main action for each character. Wouldn't it be riskier to use `hero` versions?

Reply:
With the lipsync addon they can override the lip action if they need.

Reply:
Okay, so in this case the container will be unreferenced and neither the `hero` or the version will be relevant, this will be kind of emergency procedure, out of pipe (allowed but not recommended).
Therefore in the case they rely on the container system and the pipeline, I think they should use `hero` version, to avoid things like they forget to update, they work with lips, figure this is out of date, update, and do it all over again.